lib/ai/message-utils.ts

```
@@ -261,18 +261,23 @@ export async function processChatMessages(
 
   filterEmptyAssistantMessages(messages);
 
+  // Process attachments and file content for the last message
+  messages = await processMessageContentWithAttachments(
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)